docs/Developers/Newcomers_Guide.rst

```diff
@@ -79,43 +79,46 @@ What documentation was missing? What was hard to understand?
     which can help you find a suitable repository and run coala on a bigger
     project.
 
-Once you complete this, please take the time and
-`fill this form <https://coala.io/usability>`_ so we can improve this!
+Once you complete this, please take the time to
+`fill out this form <https://coala.io/usability>`_ so we can get better!
 
 Step 1. Meet the Community!
 ---------------------------
 
-To get started, the first step is to meet the community. We use gitter to
-communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
-The newcomers should ping us "Hello World" to let us know they are here
-because we care!
+To get started, you'll want to meet the community. We use gitter to
+communicate, and the helpful community there will guide you.
+Join us at `coala gitter <https://coala.io/chat>`_. Newcomers (that's you)
+should ping us with "Hello World" to let us know they are here because
+we care!
 
 **Congratulations!** You are now part of our community.
 
 Step 2. Grab an Invitation to the Organization
 ----------------------------------------------
 
 Let us know on gitter that you are interested in contributing and ask for an
-invitation to our org. This is your first step towards contributing.
+invitation to our org. This is the first step towards contributing.
 A maintainer will command ``cobot`` (our gitter bot) to invite
-you and be part of the Newcomer team.
-The invitation will be sent by mail and you will have to accept
-it to join. If you don't find the invitation, accept it `here <https://github.com/coala>`__.
+you to the Newcomer team.
+The invitation will be sent by email and you will have to accept
+it to join. If you don't see the invitation, accept it on our `team page
+<https://github.com/coala>`__.
 
 Now that you are part of our organization, you can start working on issues.
-If you are familiar with git, you can skip the next section and pick an issue.
+If you are familiar with git/GitHub, you can skip the next section and pick an
+issue.
 
 Optional. Get Help With Git
 ---------------------------
 
-We use GitHub to manage our repository. If you're not familiar with git, we
-strongly recommend following a tutorial, such as `this one <https://try.github.io/levels/1/challenges/1>`_.
+We use GitHub to manage our repository. If you're not familiar with
+git/GitHub, we strongly recommend following a tutorial, such as `this one
```

Comment:
Yeah, this looks fine. Using `git/GitHub` just express that we exclusively use GitHub which is not true.